templates.dist/estate/default_detail.php

```diff
@@ -572,9 +573,12 @@ function headingLink($url, $title)
 			?>
 
 		</div>
-		<div class="oo-similar">
-			<?php echo $pEstates->getSimilarEstates(); ?>
-		</div>
+		<?php $similar = trim($pEstates->getSimilarEstates()); ?>
+		<?php if (!empty($similar)): ?>
+    		<div class="oo-similar">
+        		<?php echo $similar; ?>
+    		</div>
+		<?php endif; ?>
 	<?php } ?>
```

Comment:
```suggestion
	<?php if (!empty($pEstates->getSimilarEstates())) { ?>
		<?php echo $pEstates->getSimilarEstates(); ?>
	<?php } ?>
```
Ich glaube das trim() braucht man an der Stelle nicht. 
Das div würde ich wie oben angemerkt auch in die Datei der similar estates verschieben, damit dort alles an einem Ort ist.
Mir werden trotzdem noch die Überschrift und der Slider ausgegeben obwohl keine vorhanden sind. Siehe https://snipboard.io/w4pvzt.jpg
Schau dir mal an wie wir das in den [WP-Websites Templates](https://github.com/onOffice-Web-Org/onoffice-modern/blob/master/onoffice-theme/templates/estate/property-similar.php#L11) gelöst haben.
Das würde ich auch in der Datei der Einheiten einsetzen.